### ISSUE TITLE
Implement better UI

### ISSUE THREAD

**Issue by mmozeiko** [mmozeiko/pkgi#1]
* Fancy UI mode - show items in small rectangles with images. If possible to get description, allow to click on it and get more information in popup?
* Show type of game (Arcade, Strategy, etc), allow filtering/sorting on it.
* Ask to resume download or restart from beginning instead of defaulting to resume.

**Comment by chronoss09** [comment]
For me, "Show type of game" is not necessary and resume/restar game will be very usefull..., clik on it to get more information in popup is not god and this is my UI without download queue
![gui nps browser 2](https://user-images.githubusercontent.com/17669980/32030494-2c3e27be-b9f3-11e7-8af8-6b7f38214d2e.png)
This is just an exempl not the final UI...

**Comment by kenshin088** [comment]
I think you should combine all similarly named games. It's too redundant right now and makes finding other games harder when scrolling through. If there are multiple version of the game, just show one title and then once clicked, user is presented with all available regions to select from to download. then pressing O button to go back to previous list of all the games.

Only problem is with those games that have small deviation in name like a : instead of ; in the title

**Comment by mmozeiko** [comment]
What is "similar named" game? How similar? Are OlliOlli and OlliOlli2 similary named?

I'm pretty sure I don't want to fix : vs ; issues in game names or other issues like that at runtime. That is a problem with pkgi.txt file - fix it there.

**Comment by kenshin088** [comment]
I should of said exact title, not similar. Excluding sequels. 

For example there are 3 "Child of Light" games listed, one for each region. If you combined them it would just show "Child of Light" once on the list and when a user selects the game, it shows all three versions and just select the one you want. It would declutter the main screen. You can even add a little indicator next to the title to let users know there are multiple versions of the game. Or you can add something like EJU (europe/japan/usa) if those are the three available or just EJ if europe and jpn are available, etc. basically truncate the 3 letters and just add the first letter for each region or do something else to indicate multiple regions available.

for the ; and : thing, i dont think its a 'issue', it's just how the game was named. for example

(USA) Attack on Titan - Wings of Freedom
(EUR) Attack on Titan: Wings of Freedom

If you were to combine the games, you might miss these two games because of the small differences in punctuation

**Comment by mmozeiko** [comment]
Yeah those need to be fixed at the source - pkgi.txt file, not in code. Otherwise there will too many differences / typos to handle.

Does filtering out regions you don't want doesn't solve your use case about seeing game only once in the list?

**Comment by kenshin088** [comment]
It doesn't solve it imo. Some games are released in certain regions other than what I usually go for (i.e. USA). So if I wanted game X but it was only released on EU and JP, I might miss it if I was searching only for USA games. Having them all neatly together under one title ensures that I dont miss a game just because it isn't in my preferred region, but helps declutter/streamlines the UI.

I would say a large majority of people dont necessarily care about the region so long as it supports their language (unless DLC will only work for that region, then yes a person might get picky). I like having all the regions turned on but dont necessarily enjoy seeing one page of games consumed by 4 or 5 similarly/exactly named titles. By grouping them it improves the UX because you can see more unique game per page as you're trying to find the next game to download.

just my 2cents

**Comment by kenshin088** [comment]
![image](https://user-images.githubusercontent.com/33103311/32248213-4cfd9086-be5b-11e7-83e3-09e53662ccee.png)

Possible UI design by @shaffervanity. Thought I'd repost it here for them.

**Comment by chronoss09** [comment]
Greeeeeeeeeeeeeeeeatttttt !

**Comment by MarioMasta64** [comment]
this is nice and all but we would need something of a title database like how wii has with all its banner art and would require alot of loading tome for every title i dont know how much the vita can handle but id imagine it could be added as optional

**Comment by Scrxtchy** [comment]
lot of detail can be moved into submenus on the images provided.
You don't need to throw everything onto the main screen.

Compare to another software available on the 3DS. Many features are segmented into their own screens. The system gets away with having multiple screens available at once since it has two displays.

Search, Description and Cover art are fine.
Download queue can be moved elsewhere
Filters work as a overlay/popup menu. These will help navigation controls
Storage space can be removed most likely. I wonder if it's possible to get an accurate estimate of the total download size. If so we can display a warning message for how much space would be required. As it stands, having it as a percentage meter means absolutely nothing
